content/en/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md

```diff
@@ -9,6 +9,15 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.28"
+    toVersion: "1.30"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
 ---
-Allow the API server to serve consistent lists from cache.
+Enhance Kubernetes API server performance by serving consistent list requests
+directly from its watch cache, improving scalability and response times.
+This feature requires a newer etcd version (v3.4.33+ or v3.5.13+) and ensures
+data consistency while reducing the need for resource-intensive quorum reads from etcd.
+
+See the Kubernetes documentation on [Semantics for get and list](/docs/reference/using-api/api-concepts/#semantics-for-get-and-list) for more details.
```

Comment:
See the Kubernetes documentation on [Semantics for **get** and **list**](/docs/reference/using-api/api-concepts/#semantics-for-get-and-list) for more details.